.changeset/lovely-dolls-taste.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": minor
```

Comment:
Not really a new feature, it's a bug fix, so I think we can go with a patch bump here.